pkg/remediator/watch/filteredwatcher.go

```diff
@@ -329,6 +348,8 @@ func (w *filteredWatcher) start(ctx context.Context, resourceVersion string) (bo
 	if err != nil {
 		if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
 			return false, status.InternalWrapf(err, "failed to start watch for %s", w.gvk)
+		} else if apierrors.IsNotFound(err) {
+			metrics.RecordResourceConflict(ctx, w.getLatestCommit())
```

Comment:
Should we also wrap the err here with ConflictWatchResourceDoesNotExist?

Reply:
Done